python-main.js

```diff
@@ -1178,6 +1182,18 @@ function web_editor(config) {
             // Stop immediate closure
             e.stopImmediatePropagation();
         });
+
+        window.onresize = function(){
```

Comment:
Normally I would add event handlers using `window.addEventListener('_event_', _callback_)`. Is there any advantage of using this method? I haven't seen this one before, so I wonder if that would replay any other handlers applied.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event#Specifications

Looking over some docs, it seems the `window.onresize` is equivalent to `window.addEventListener('_event_', _callback_)`, so from what I understand I don't think there are any advantages/disadvantages. I can just use the `addEventListener()` method to keep event handling the same throughout the code if you'd like.